ops-bedrock/docker-compose.yml

```diff
@@ -262,18 +264,20 @@ services:
     depends_on:
       orchestrator:
         condition: service_healthy
-    extra_hosts:
-      - "host.docker.internal:host-gateway"
```

Comment:
This is needed by the e2e tests, which use host networking for `ESPRESSO_SEQUENCER_L1_PROVIDER`. That will be fixed by @nomaxg 's change though to separate the docker-compose files